additions (#152)
fiskaltrust/faq: qna/General-entitlement-primary-contact-en.md

General-entitlement-primary-contact-ENG.md
## Question

What is the Primary Contact and how can I change it?

## Metadata tags

lang-eng, market-de, shop, PosCreator, PosDealer, Consultant

## Answer

> With the registration of your company in the fiskaltrust.portal on the one hand an account and on the other hand a contact is created. This contact has as primary contact every possible authorization in the account of the company.

Reviewer: ```suggestion
With the registration of your company in the fiskaltrust.Portal on the one hand an account and on the other hand a contact is created. This contact has as primary contact every possible authorization in the account of the company, it is the so-called administrator.
```

If you create additional contacts under <kbd>Company name</kbd> &rarr; <kbd>Employee</kbd>, you can click on the record and assign specific permissions. The highlighted record is the one with the primary role.
Below the permissions you will further find the <kbd>Primary Contact</kbd> button, which you can use to set a contact as the new primary contact.

> By changing the primary contact, all permissions are released for the selected contact and your permissions are completely disabled. This means that with your contact **no login in the portal** is possible anymore. The new primary contact can assign permissions to your contact again, if desired.

Reviewer: ```suggestion
By changing the primary contact, all permissions are released for the selected contact and your permissions are completely disabled. This means that with this contact **no login in the portal** is possible anymore, for this at lease read permission is necessary. The new primary contact can assign permissions to this contact again, if desired.
```

> By changing the primary contact, all permissions are released for the selected contact and your permissions are completely disabled. This means that with this contact **no login in the portal** is possible anymore, for this at lease read permission is necessary. The new primary contact can assign permissions to this contact again, if desired.

Reviewer 2: ```suggestion
By changing the primary contact, all permissions of the previous primary account are revoked and its account owner will not be able to log on to the fiskaltrust.Portal any longer, as that action requires the read permission as minimum. The new primary contact may assign permissions to this contact again, if desired.
```
